packages/web/src/components/leaderboard/leaderboard-table/mobile-leaderboard-table/MobileLeaderboardTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { MOBILE_LEADERBOARD_TD_WIDTH } from "@constants/skeleton.constant";
+import { LeaderModel } from "@models/leaderboard/leader-model";
+import MobileLeaderboardTableRow from "../../mobile-leaderboard-table-row/MobileLeaderboardTableRow";
+
+export default function MobileLeaderboardTable({
+  me,
+  leaders,
+  isMobile,
+}: {
+  me?: LeaderModel;
+  leaders: LeaderModel[];
+  isMobile: boolean;
+}) {
+  return (
+    <>
+      {me && (
+        <MobileLeaderboardTableRow
+          item={me}
+          tdWidths={MOBILE_LEADERBOARD_TD_WIDTH}
```

Comment:
This constant value is imported from the skeleton definition file. 
Check out `common/values`, which manages static variables.

Reply:
I kept constant values for table widths and skeleton structures In the file, such as `LEADERBOARD_TD_WIDTH`, `MOBILE_LEADERBOARD_TD_WIDTH`, `TABLET_LEADERBOARD_TD_WIDTH`, `LEADER_INFO`, and so on. because I’ve noticed that’s where other devs in the past and Khiem kept such variables as well. 

However, I think it would be nice to move all those variables into its own separate file like you pointed out. though, we probably need to talk about it with Khiem and also tidy up code from past devs